content/en/blogs/Kubernetes-multicluster-KubeSphere.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Manage Multiple Clusters and Deploy Applications for Kubernetes in Hybrid Cloud'  
```

Comment:
I prefer this title: Kubernetes Multi-cluster management and application deployment in Hybrid Cloud.